find_nearer_stores/find_nearer_stores.py

```diff
@@ -0,0 +1,331 @@
+# -*- coding: utf-8 -*-
+
+import math
+import os
+import pandas as pd
+import re
+import requests
+from urllib.parse import urlencode
```

Comment:
Optimize import statements (in IntelliJ, use shortcut <kbd>CTRL</kbd> + <kbd>ALT</kbd> + <kbd>O</kbd>, O=Optimize).